.gitignore

```diff
@@ -1,4 +1,4 @@
 out/
 node_modules/
-.vscode/
-.vscode-test/
\ No newline at end of file
+.vscode-test/
+package-lock.json
```

Comment:
package-lock.json is intended to be checked into source control as far as I know